src/LodeRunner.API.Test/IntegrationTests/Controllers/LoadTestConfigs.cs

```diff
@@ -146,6 +162,23 @@ public async Task CanPutLoadTestConfigs()
             Assert.Equal(JsonSerializer.Serialize(expectedLoadTestConfig), JsonSerializer.Serialize(actualLoadTestConfig));
         }
 
+        /// <summary>
+        /// Determines whether this instance [can update loadTestConfig with an improperly formatted identifier].
+        /// </summary>
+        /// <returns><see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        [Trait("Category", "Integration")]
+        public async Task CannotPutLoadTestConfigsWithInvalidId()
```

Comment:
Do we need CannotPost use case?

Reply:
I left it out because PUT calls the POST method, and all the id validation in PUT is actually done in the resultHandler's POST method

Reply:
Added CannotPostInvalidLoadTestConfig